googlesource/source_intergration_test.go

```diff
@@ -121,16 +128,107 @@ func cleanupDataSet() (err error) {
 	}
 	defer client.Close()
 
-	table := client.Dataset(datasetID).Table(tableID)
-	if err := table.Delete(ctx); err != nil {
+	meta := &bigquery.DatasetMetadata{
+		Location: location, // See https://cloud.google.com/bigquery/docs/locations
+	}
+
+	// create dataset
+	if err := client.Dataset(datasetID).Create(ctx, meta); err != nil && !strings.Contains(err.Error(), "duplicate") {
 		return err
 	}
+	fmt.Println("Dataset created")
```

Comment:
I'd suggest using `t.Log()` if we need to log something in tests. Similar for other places.